payjoin/src/receive/v2/session.rs

```diff
@@ -437,6 +479,19 @@ mod tests {
                 state: PayjoinProposal { v1: payjoin_proposal, context: session_context },
             }),
         };
-        run_session_history_test(test);
+        run_session_history_test(test)
+    }
+
+    #[test]
+    fn test_session_history_uri() -> Result<(), BoxError> {
+        let session_context = SHARED_CONTEXT.clone();
+        let events = vec![SessionEvent::Created(session_context.clone())];
+
+        let uri = SessionHistory { events }.pj_uri().expect("Could not get session history uri");
```

Comment:
personal nit: with expects I think its idomatic to explain why we expect a certain condition to be true rather than an error message. i.e expect("session creation event is present")